source/adios2/toolkit/transport/file/FilePointer.cpp

```diff
@@ -48,16 +48,16 @@ void FilePointer::Open(const std::string &name, const OpenMode openMode)
 
     if (m_OpenMode == OpenMode::Write)
     {
-        m_File = std::fopen(name.c_str(), "w");
+        m_File = fopen(name.c_str(), "wb");
```

Comment:
I tried #define _CRT_SECURE_NO_WARNINGS on my Windows 10 and Visual Studio 17. It would still give the warning for fopen and ctime. C++ ifstream binary at worse is a thin wrapper around C stdio + 1 or 2 pointer redirections for write(buffer, size), at best it will go deeper into the kernel IO implementation. I've never experience any slowdown as both will pick up the system buffer size. The bad reputation comes from the fstream as a stringstream where the overloaded << >> for casting types will slow things down in heap, but not for system IO writes and reads (which are probably orders of magnitude larger).